dataverse/orgsvc/C#/RemoveUnrefrencedQueues/RemoveUnrefrencedQueues/RemoveUnrefrencedQueues.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Client;
+using System.Threading.Tasks;
+using Microsoft.Xrm.Sdk.Query;
+using System.ServiceModel;
+using Microsoft.Crm.Sdk.Messages;
+using System.Threading;
+using System.Collections.Generic;
+using Microsoft.Xrm.Sdk.PluginTelemetry;
+using System.Linq;
+using System.Diagnostics;
+using System.Linq.Expressions;
+
+namespace PowerApps.Samples
+{
+    /// <summary>
+    /// The plug-in creates a task activity after a new account is created. The activity reminds the user to
+    /// follow-up with the new account customer one week after the account was created.
+    /// </summary>
+    /// <remarks>Register this plug-in on the Create message, account entity, and asynchronous mode.
+    /// </remarks>
+
+    public class RemoveUnrefrencedQueues : IPlugin
```

Comment:
I would prefer that this plug-in use the PluginBase class that is generated when using the Visual Studio extension or pac plugin init. But we have a lot of existing samples that still implement IPlugin directly.  

Also, we are encouraging people to use plug-in packages for all new plugins so that they can include dependent assemblies as needed. See [Create and register a plug-in package using PAC CLI](https://learn.microsoft.com/en-us/power-platform/developer/howto/cli-create-package)

That isn't the purpose of this sample, so I'll ask @phecke  to comment on this.

Reply:
I lean towards it's not the purpose of this sample and that will add a lot of code for the user to understand how it all works together.  I know it was really confusing for me.